includes/validation/class-amp-validation-manager.php

```diff
@@ -201,42 +213,45 @@ class AMP_Validation_Manager {
 	 */
 	protected static $stylesheet_slug;
 
+	/**
+	 * Get dev tools user access service.
+	 *
+	 * @return DevToolsUserAccess
+	 */
+	private static function get_dev_tools_user_access() {
+		/** @var DevToolsUserAccess $service */
+		$service = Services::get( 'dev_tools.user_access' );
+		return $service;
+	}
+
 	/**
 	 * Initialize.
 	 *
 	 * @return void
 	 */
 	public static function init() {
+		add_filter( 'map_meta_cap', [ __CLASS__, 'map_meta_cap' ], 9, 2 );
```

Comment:
Better to filter later perhaps to make it easier for plugins to override at lower priority.

```suggestion
		add_filter( 'map_meta_cap', [ __CLASS__, 'map_meta_cap' ], 100, 2 );
```